src/IdentityManager2/IdentityManager2.csproj

```diff
@@ -37,8 +37,6 @@
 
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.0" />
-    <PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="3.1.0" />
```

Comment:
*Microsoft.Extensions.FileProviders.Embedded* this is also provided by *Microsoft.AspNetCore.App*, no need to duplicate